### PR TITLE
fixed user auth broken links and skygear error sample codes

### PR DESCRIPTION
**Pull request by xiuwenlu**

### CODE REVIEW

**Comment by oursky-travis** [comment]
@xiuwenlu, thanks for your PR! By analyzing the history of the files in this pull request, we identified @Steven-Chan to be a potential reviewer.

**Comment by chpapa** [comment]
@xiuwenlu thx, first thing, remove the .DS_store in your PR?

**Comment by xiuwenlu** [comment]
@chpapa sorry, they are now removed.

File: auth/basics/js.md [hunk 1]
```diff
@@ -93,15 +93,15 @@ skygear.auth.signupWithUsername(username, password).then((user) => {
   console.log(user); // user object
 }, (error) => {
   console.error(error);
-  if (error.error.code === skygearError.Duplicated) {
+  if (error.error.code === skygear.ErrorCodes.Duplicated) {
```
Comment (Steven-Chan) [inline auth/basics/js.md:95]: If `skygear.ErrorCodes` is the correct way to get the error codes, then I guess `import skygearError from 'skygear/lib/error';` is not needed in the previous lines.

Same for other error codes example code update.

**Review by Steven-Chan — CHANGES_REQUESTED** [review]

File: auth/basics/js.md (hunk 1 above)
Comment (Steven-Chan) [inline auth/basics/js.md]: Wait, I just found https://github.com/SkygearIO/guides/pull/81.

Comment (xiuwenlu) [reply auth/basics/js.md:95]: Oh, I'm not totally sure how it works for node.js config, but I know that for CDN, we need to use skygear.ErrorCodes instead.

**Comment by tensiuyan** [comment]
ref #82

File: auth/basics/js.md (hunk 1 above)
Comment (Steven-Chan) [reply auth/basics/js.md:95]: You may just take away `import skygearError from 'skygear/lib/error';`.

Because `skygearError` is no longer used.

Comment (Steven-Chan) [reply auth/basics/js.md:95]: You may find that in line 90 https://github.com/SkygearIO/guides/pull/85/files#diff-d2054f012eb664511c6d732f75ff5d76R90.

Comment (xiuwenlu) [reply auth/basics/js.md:95]: @Steven-Chan it's removed now.

**Review by Steven-Chan — APPROVED** [review]

**Pull request merged**